tools/vim/Readme

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2016 Marcin Copik
+#
+#  Distributed under the Boost Software License, Version 1.0. (See accompanying
+#  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+Automatically export configuration file for VIM plugin YouCompleteMe: https://github.com/Valloric/YouCompleteMe
+
+Installation:
+https://github.com/Valloric/YouCompleteMe
+
+Configuration:
+Configure hpx build, passing location for YCM configuration file with option HPX_WITH_VIM_YCM. File will be configured to use HPX compilation flags and placed in this directory.
+Configuration file should be placed in your working directory or in any directory above. An optimal choice may be the source directory.
+To avoid asking for confirmation and automatically load config file add this information to your .vimrc file: 
+let g:ycm_extra_conf_globlist = ['LOCATION_OF_CONFIG_FILE/*']
+
+YCM uses compilation flags exported by CMake in build directory. Do not remove compile_commands.json!
```

Comment:
There is no newline at the end of the file. Also, please bline break the text at ~80 columns. For some reason this file is not being checked by inspect, so this didn't show up on CircleCI.